src/main/java/com/netgrif/application/engine/petrinet/domain/repositories/PetriNetRepository.java

```diff
@@ -26,5 +26,7 @@ public interface PetriNetRepository extends MongoRepository<PetriNet, String>, Q
 
     List<PetriNet> findAllByVersion(Version version);
 
+    List<PetriNet> findAllByUriNodeId(String uri);
```

Comment:
paged alternative? How large do we expect these collections to be?

Reply:
It is not expected to have a large number of processes on one URI. So I think a paged alternative is not necessary.